### PR TITLE
cmd: Implement copy small file

### PR DESCRIPTION
**Pull request by Prnyself**

### CODE REVIEW

**Comment by Xuanwo** [comment]
We need a BCP to go futher.

File: operations/cp.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
+package operations
+
+import (
+	"io"
+)
+
+func (do *DualOperator) Copy(src, dst string) {
```
Comment (Xuanwo) [inline operations/cp.go:7]: How about introducing a ResultChannel like https://github.com/beyondstorage/beyond-ctl/pull/26 ?

Comment (Prnyself) [reply operations/cp.go:7]: Result contains `targetObject` and `Error`?

Comment (Xuanwo) [reply operations/cp.go:7]: For `copy` here, I think `error` is enough.

```go
type Result {
    err error
}
```

Comment (Prnyself) [reply operations/cp.go]: return `err` and a `Result` only have `Err` field looks weird.

**Review by Xuanwo — APPROVED** [review]

**Pull request merged**